### ISSUE TITLE
GetConfiguration::isValid() issues warning

### ISSUE THREAD

**Issue by andriokha** [afterpay/sdk-php#13]
Hi,

I have a function for sending Afterpay requests that accepts a `\Afterpay\SDK\HTTP\Request`. It injects merchant config, checks validity, does the send, checks for certain errors and returns the response. I've noticed a warning when sending a `GetConfiguration` request coming from a call to `$request->isValid()` (ie `\Afterpay\SDK\Shared\ModelMethods::isValid()`).

```
Notice: Undefined property: Afterpay\SDK\HTTP\Request\GetConfiguration::$data in Afterpay\SDK\HTTP\Request->getValidationErrors() (line 291 of /var/www/html/env/vendor/afterpay-global/afterpay-sdk-php/src/Shared/ModelMethods.php).
Afterpay\SDK\HTTP\Request->getValidationErrors() (Line: 281)
Afterpay\SDK\HTTP\Request->isValid() (Line: 270)
```
As far as I can tell this is because `GetConfiguration` has no body parameters and therefore the class has no `$data` property, unlike most other requests.

I was wondering if it might make sense to add `$data` as an empty array, either to `Afterpay\SDK\HTTP\Request\GetConfiguration` directly or `\Afterpay\SDK\HTTP\Request`? From my PoV as a consumer of the library it would be great to be able to call `isValid()` on any request, for simpler code.

Also, although it's only the one request that's been causing me a problem (because it's a fairly minimal integration, at this point at least), looking through the codebase it looks like I'd have similar problems with:
- `\Afterpay\SDK\HTTP\Request\GetCheckout`
- `\Afterpay\SDK\HTTP\Request\Ping`

Thanks in advance!

**Comment by benfterpay** [comment]
Hi @andriokha,

Thanks for your message. This is a really great pickup, thanks for raising it. The use case for checking if a `GET` request is valid perhaps had not been considered. One solution may be to simply check that `$data` exists before trying to inspect it.

This will be discussed within the team. First impression is that it seems like a simple fix. Please expect another response within the next few days.

Thanks for your patience.

**Comment by benfterpay** [comment]
Hi @andriokha,

This has been resolved in #17. The issue has been automatically closed, but it'd be great if you can verify the resolution. If not satisfied with the fix, please feel free to reopen the issue.

Thank you.
